net-dns/yaddns/yaddns-1.1.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="A small dynamic dns client with multiple servers support and a high flexibility"
+DESCRIPTION="Yaddns is a small dynamic dns client with multiple servers support and a high flexibility."
```

Comment:
Please wrap this line as required by the [Guidelines][0]

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#general-tips